.kokoro

```diff
@@ -407,6 +408,12 @@ run_cocoapods() {
       bash scripts/test_all catalog/MDCCatalog.xcworkspace:MDCCatalog
     fi
   elif [ "$DEPENDENCY_SYSTEM" = "cocoapods-podspec" ]; then
+    gem install cocoapods-deintegrate
```

Comment:
Please also wrap the new gem/pod commands in a `if [ -n "$KOKORO_BUILD_NUMBER" ]; then` block so it doesn't mess around with our development environments.

Reply:
Again, this wasn't meant to be looked at. sorry for the spam!